KitePrintSDK/src/main/java/ly/kite/print/AssetUploadRequestListener.java

```diff
@@ -7,6 +9,6 @@
  */
 interface AssetUploadRequestListener {
     void onProgress(AssetUploadRequest req, int totalAssetsUploaded, int totalAssetsToUpload, long bytesWritten, long totalAssetBytesWritten, long totalAssetBytesExpectedToWrite);
-    void onUploadComplete(AssetUploadRequest req, List<Asset> assets);
+    void onUploadComplete(Context context, AssetUploadRequest req, List<Asset> assets);
```

Comment:
I'm curious, do we need to pass out the context here? Feels wrong from a public interface point of view.